confgenerator/logging.go

```diff
@@ -35,9 +35,11 @@ func setLogNameProcessor(ctx context.Context, logName string) LoggingProcessorMo
 		"logName": {
 			DefaultValue: &logName,
 		},
-		`labels."compute.googleapis.com/resource_name"`: {
+	}
+	if hostName != "" {
+		fields[`labels."compute.googleapis.com/resource_name"`] = &ModifyField{
```

Comment:
Note : This change (i don't fully remember if this was intended to be kept or not) adds a lot of `golden` updates.